docs/DeveloperGuide.md

```diff
@@ -8,22 +8,107 @@
 
 {Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
 
+### _Designs_
+### 1. Calories Management Mode
+![Commands for managing calories](./diagrams/CaloriesManagement.png)
+   * when `caloriesManagement` is executed in Parser, different actions will be carried out based on the commands.
+   * if `eat`, store the name and calories of the input food
+   * if `view`, display all the foods consumed
```

Comment:
Instead of "if eat" and "if view", perhaps it would be clearer to mention which variable fulfils those conditions. It could be phrased as "if command == eat" and "if command == view".